DEVELOPMENT.md

```diff
@@ -379,6 +379,177 @@ Comments can also be used to indicate steps to take in the future (*TODOs*). Suc
 
 In our API definitions (`.proto` files) we'd like to see short comments on every service, method, message and field. Code that is generated from these files does not have to comply with guidelines (such as Go's guideline for starting the comment with the name of the thing that is commented on).
 
+## JavaScript (Frontend) Code Style
+
+For our frontend development, we use JavaScript ES6, which is either transpiled by `webpack` to be run in the browser, or used natively by Node.JS in our build pipeline.
```

Comment:
> used natively by Node.JS in our build pipeline. - natively doesnt really seem to be a proper word to describe that we do not transpile js code to run in the node.js environment. Besides that, we actually do transpile all js code (sdk/webpack configs/webui)

True. I'll scratch the whole part.

> we use JavaScript ES6 - i guess you use ES6 to say that we use the 'modern' version of javascript, however this can be confusing as we extensively use features from the later standards

Do you have a suggestion on how to word this better?